ast/dml.go

```diff
@@ -1558,7 +1558,159 @@ type ShowStmt struct {
 
 // Restore implements Node interface.
 func (n *ShowStmt) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	RestoreOptFull := func() {
```

Comment:
Better use lower-case here, following the naming convention of local variables.

```suggestion
	restoreOptFull := func() {
```